src/components/TechTalk/TechTalk.js

```diff
@@ -0,0 +1,141 @@
+import React, {Fragment} from 'react';
+import {Box, Grid, Typography} from '@material-ui/core';
+import {makeStyles, useTheme} from '@material-ui/core/styles';
+import {Link} from 'gatsby';
+import PropTypes from 'prop-types';
+import ResponsiveVideo from '../shared/ResponsiveVideo';
+import Tag from '../shared/Tag';
+
+const useStyles = makeStyles((theme) => ({
+  sectionTitle: {
+    padding: theme.spacing(3, 0, 6),
+  },
+  sectionContainer: {
+    paddingBottom: theme.spacing(8),
+  },
+  paragraphContainer: {
+    paddingBottom: theme.spacing(4),
+  },
+  surveyButton: {
+    textTransform: 'none',
+    verticalAlign: 'top',
+    lineHeight: 1.5,
+    fontFamily: 'Varela, Arial, sans-serif',
+  },
+}));
+
+const TechTalk = ({techTalk}) => {
+  const classes = useStyles();
+  const theme = useTheme();
+
+  // Split the description into different paragraphs based on new lines
+  const descriptionParagraphs = techTalk.frontmatter.description.split(
+    /\r?\n\n/
+  );
+
+  return (
+    <Box m="auto" maxWidth={theme.breakpoints.values.lg}>
+      <Box className={classes.sectionContainer}>
+        <Typography variant="h1" className={classes.sectionTitle}>
+          {techTalk.frontmatter.title}
+        </Typography>
+        {descriptionParagraphs.map((paragraph, key) => (
+          <Typography
+            variant="body1"
+            key={key}
+            className={classes.paragraphContainer}
+          >
+            {paragraph}
+          </Typography>
+        ))}
+      </Box>
+
+      <Box className={classes.sectionContainer}>
+        <Typography variant="h2" className={classes.sectionTitle}>
+          Speakers
+        </Typography>
+        <ul>
+          {techTalk.frontmatter.speakers.map((speaker) => (
+            <Typography
```

Comment:
And maybe replace this with a Material List Item component too

Reply:
Will do!